x-pack/plugins/observability/public/components/shared/exploratory_view/header/add_to_case_action.tsx

```diff
@@ -31,11 +32,14 @@ export function AddToCaseAction({ lensAttributes, timeRange }: AddToCaseProps) {
     [http.basePath]
   );
 
+  const absoluteFromDate = parseRelativeDate(timeRange.from);
+  const absoluteToDate = parseRelativeDate(timeRange.to, { roundUp: true });
+
   const { createCaseUrl, goToCreateCase, onCaseClicked, isCasesOpen, setIsCasesOpen, isSaving } =
     useAddToCase({
       lensAttributes,
       getToastText,
-      timeRange,
+      timeRange: { from: absoluteFromDate.toISOString(), to: absoluteToDate.toISOString() },
```

Comment:
Similarly, here I think we could we could add a test to ensure that these are passed as absolute dates to `addToCase`. In the test we could either check for the correct absolute dates on the URL (which would, in my opinion, be the most straightforward and decoupled option), or we could check the parameters passed to the hook itself (which I don't really like as it tightly couples the test with the implementation).

Reply:
@lucasfcosta added a test case [`274ebac` (#117487)](https://github.com/elastic/kibana/pull/117487/commits/274ebaccd9353a0c8e127c77dc782375aeb5352b)